src/dataprotection/azext_dataprotection/manual/custom.py

```diff
@@ -30,6 +31,7 @@
     QueryRequest, QueryRequestOptions
 from azext_dataprotection.manual import backupcenter_helper, helpers as helper
 from azext_dataprotection.aaz.latest.dataprotection.backup_vault import Show as BackupVaultGet
+# from azext_dataprotection.aaz.latest.dataprotection.backup_vault import Update as BackupVaultUpdate
```

Comment:
As the new lines are commented out, just remove them entirely.

Reply:
resolved

Reply:
resolved